packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -214,6 +219,27 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
   /// The nodes being kept alive despite not being visible.
   final Map<int, RenderBox> _keepAliveBucket = <int, RenderBox>{};
 
+  List<RenderBox> _debugDanglingKeepAlives;
+
+  /// Indicates whether child order verification is enabled
```

Comment:
also this needs to define what "child order verification" means

also this seems to do more than that? even when it's set to false this still calls _debugVerifyChildOrder. When it's true is also checks a dangling keep alive list?

Reply:
_debugVerifyChildOrder will check if it is enabled before doing anything